package.json

```diff
@@ -22,15 +101,27 @@
     }
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
     "lint": "eslint ./",
-    "test": "yarn lint && yarn jest",
-    "jest": "react-scripts test --env=jsdom --coverage"
+    "test": "yarn lint && yarn jest  --watch",
```

Comment:
you shouldn't change this things

Reply:
I only add watch to avoid strating this all the time